source/numerics/error_estimator.cc

```diff
@@ -954,7 +954,11 @@ estimate (const Mapping<dim, spacedim>      &mapping,
           const unsigned int       n_threads,
           const types::subdomain_id subdomain_id,
           const types::material_id       material_id,
+#ifdef DEAL_II_MSVC
+          const typename Strategy strategy)
+#else
```

Comment:
No need for the `typename` here (I suppose, not knowing MSVC). It is in fact wrong in C++98.